jaseci_kit/jaseci_kit/modules/object_detection/requirements.txt

```diff
@@ -0,0 +1,46 @@
+# YOLOv5 requirements
+# Usage: pip install -r requirements.txt
+import sys
+sys.path.insert(0, './yolov5')
```

Comment:
Is this necessary? Does it even work inside requirements.txt?

Reply:
This is what I got from the documentation from yolo. requirements.txt file is only for pip install and not to insert sys.oath